sites/kit.svelte.dev/src/lib/search/SearchBox.svelte

```diff
@@ -329,8 +325,22 @@
 		padding: 1rem;
 	}
 
+	:root {
+		--sk-search-bg-hover: hsl(0, 0%, 93%);
+		--sk-search-small-focus: hsla(0, 0%, 100%, 0.6);
+		--sk-search-strong-focus: hsl(0, 0%, 100%);
```

Comment:
I think we should try to reuse the existing tokens as much as possible. The third var for example exists as `--sk-back-1` for light mode already. It doesn't have to be exactly like the previous version if it means we can reuse existing tokens.

Reply:
Do you mean something like this?
```
:root {
    --sk-search-small-focus: var(--sk-back-4);
    --sk-search-strong-focus: var(--sk-back-2);
}
```
There aren't any tokens that are white in both light and dark mode. Should there be? Or is it ok to use a raw colour value?